tests/integration/ambient/baseline_test.go

```diff
@@ -1441,13 +1606,109 @@ spec:
 							from.CallOrFail(t, echo.CallOptions{
 								Address: "111.111.222.222",
 								Port:    to.PortForName("http"),
+								// we want to make sure the workload selector labels are honored, so we assert the response contains either uncaptured-v1 or uncaptured-v2
+								// sample response:
+								//
+								// ServiceVersion=v1
+								// ServicePort=8080
+								// Host=serviceentry.istio.io
+								// URL=/any/path
+								// Cluster=cluster-0
+								// IstioVersion=
+								// Method=GET
+								// Proto=HTTP/1.1
+								// IP=10.244.2.20
+								// Alpn=
+								// RequestHeader=Accept:*/*
+								// RequestHeader=User-Agent:curl/7.81.0
+								// Hostname=uncaptured-v1-868c9b59b5-rxvfq
+								Check: check.BodyContains(`Hostname=uncaptured-v`),
```

Comment:
nit; stronger assertion to have both v1 +v2 checks serially to ensure we are load balancing

Reply:
addressed